SYCL/DeviceLib/imf_simd_emulate_test.cpp

```diff
@@ -0,0 +1,1835 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// RUN: %clangxx -fsycl -fno-builtin -fsycl-device-lib-jit-link %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+#include <CL/sycl.hpp>
+#include <cassert>
+#include <iostream>
+
+namespace s = cl::sycl;
```

Comment:
There is ongoing activity to remove 'cl'. Please change it same way as it is done here: https://github.com/intel/llvm-test-suite/blob/intel/SYCL/DeviceLib/math_test.cpp#L15
```suggestion
namespace s = sycl;
```